### ISSUE TITLE
Can't launch .jar on Ubuntu

### ISSUE THREAD

**Issue by danielms215** [AurousGroup/Bug-Tracker#29]
Error message "Could not find or load main class me.aurous.Aurous".
Java 8 and dependencies installed.

**Comment by feeloow** [comment]
Same here with java-8-openjdk-amd64 on ubuntu 15.04

**Comment by gregor160300** [comment]
I have another error: gregor@gregor-Xubuntu:~/Downloads/Aurous$ sudo java -jar Aurous.jarJava HotSpot(TM) 64-Bit Server VM warning: You have loaded library /home/gregor/Downloads/Aurous/linux/libjxbrowser-common.so which might have disabled stack guard. The VM will try to fix the stack guard now.
It's highly recommended that you fix the library with 'execstack -c <libfile>', or link it with '-z noexecstack'.
Exception in thread "IPC Process Thread" java.lang.UnsatisfiedLinkError: /home/gregor/Downloads/Aurous/linux/libjxbrowser-common.so: /home/gregor/Downloads/Aurous/linux/libjxbrowser-common.so: wrong ELF class: ELFCLASS32 (Possible cause: architecture word width mismatch)
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1938)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1821)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at com.teamdev.jxbrowser.chromium.internal.LibraryLoader.loadLibrary(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.SharedMemoryLibrary.<init>(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.SharedMemoryLibrary.getInstance(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.ChromiumProcess.start(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.d.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

**Comment by gregor160300** [comment]
Xubuntu 15.04, I followed the readme

**Comment by Boltgolt** [comment]
@gregor160300 i had the same problem, turns out to be because you used a 64-bit version of java. see #5

**Comment by JulienMEME** [comment]
Hi, 
I have the same problem with danielms215 and feeloow :
"Could not find or load main class me.aurous.Aurous" on openSUSE 13.2"

gcc, libgcrypt11 and libgcrypt11-debuginfo (instead of libgcrypt11-dev) was installed (2 last not official).

openjdk version "1.8.0_60"
OpenJDK Runtime Environment (build 1.8.0_60-b27)
OpenJDK 64-Bit Server VM (build 25.60-b23, mixed mode)

Perhaps Oracle's versions mandatory?

**Comment by EndySlay** [comment]
If You Have Java 8 run the command in the terminal (ubuntu and its derivatives only)
sudo java -jar Aurous.jar 
then type in your password it should launch.the devs are working on ways to launch it without using  sudo. to add this app to unity dash  i suggest using menulibre

**Comment by EndySlay** [comment]
also i didnt use the openjdk version of java which may be a issue i used oracles own version which you can get by running these commands
sudo add-apt-repository ppa:webupd8team/java
sudo apt-get update
sudo apt-get install oracle-java8-installer

**Comment by dyskette** [comment]
@EndySlay Yeah, you are probably fixing another issue with sudo. Because on Fedora 23, using sudo does not fix anything. It just print the same message.

```
Error: Could not find or load main class me.aurous.Aurous
```

It could probably be openjdk, but I won't change the java version just for one program. If this is the case, then it's a bug.

**Comment by Lucasvo1** [comment]
Same issue here on Fedora 22
